.github/workflows/deploy-admin-app-s3.yml

```diff
@@ -0,0 +1,42 @@
+name: Admin app S3 deployment
+on:
+  push:
+    branches:
+      - deploy-admin-app
```

Comment:
Don't forget to change it back to `main` 🙂 
I recommend adding this task to your checklist 🙂